.editorconfig

```diff
@@ -221,6 +221,9 @@ csharp_preserve_single_line_statements = true
 # warning RS0037: PublicAPI.txt is missing '#nullable enable'
 dotnet_diagnostic.RS0037.severity = none
 
+# RS0046: Avoid the 'Opt' suffix
```

Comment:
File an issue to clean up?

Reply:
yup, filed, thanks for the reminder.
https://github.com/dotnet/roslyn/issues/45705